eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```diff
@@ -98,6 +98,9 @@ services:
         class: Liip\ImagineBundle\Imagine\Cache\Resolver\ResolverInterface
         factory: 'ezpublish.image_alias.imagine.cache_resolver_decorator_factory:createCacheResolver'
         decorates: ezpublish.image_alias.imagine.cache_resolver
+        # NB! Depends on https://github.com/symfony/symfony/pull/31551 which is not going to be added to 2.8!!
+        # We probably need to make sure class is a (common) class to workaround this on 6.x
+        lazy: true
```

Comment:
@kmadejski Seems this needs to set some class as class for 6.x in order for this to be lazy.
Any suggestions?

Reply:
@andrerom I set this service to lazy and checked it on both 6.7 and 6.13. The container has been recreated without any error or warning and the feature works correctly. Assuming that I missed something, I think the best option will be to set `\eZ\Bundle\EzPublishCoreBundle\Imagine\Cache\Resolver\ProxyResolver` if it's indeed required to define a particular class instead of an interface though.

Reply:
> The container has been recreated without any error or warning and the feature works correctly.

You need the config resolver changes in this PR to get a warning.
W/o you can only observe that the service is not lazy, and most likely also that `image_host` setting not working as intended on command line unless you force it on default scope.

Reply:
>  I think the best option will be to set \eZ\Bundle\EzPublishCoreBundle\Imagine\Cache\Resolver\ProxyResolver if it's indeed required to define a particular class instead of an interface though.

This will break on RelativeResolver usage, best is if there is a common class they both implement
For instance if we change RelativeResolver to extend liip´s ProxyResolver.

We don't need this on 2.5 and up where the fix seems like it will be accepted in Symfony itself.

Reply:
> W/o you can only observe that the service is not lazy, and most likely also that image_host setting not working as intended on command line unless you force it on default scope.

Correct, my bad.

> For instance if we change RelativeResolver to extend liip´s ProxyResolver.

Yes, it should do the job, I think. Can you handle it as a part of this PR or would you like (me) to open another one?

> We don't need this on 2.5 and up where the fix seems like it will be accepted in Symfony itself.

Indeed, I double-checked while I've been working on this feature.

Reply:
Moved this fix to https://github.com/ezsystems/ezpublish-kernel/pull/2649
(I adapted RelativeResolver to extend LiipProxyResolver, and set that as class to allow it to be lazy)